.github/workflows/conflicts.yml

```diff
@@ -0,0 +1,26 @@
+name: 'Merge Conflict Watcher'
+on:
+  pull_request:
+    branches:
+      - main
```

Comment:
Add more event triggers, like `synchronize` and `edited` is a must.

Use this [link](https://docs.github.com/en/enterprise-server@2.22/actions/reference/events-that-trigger-workflows#pull_request)